test/utility.test.ts

```diff
@@ -179,7 +181,7 @@ suite("utility tests", () => {
         windows: null,
         test: ["test"],
       });
-      return stubMap;
+      return (stubMap as unknown) as vscode.WorkspaceConfiguration;
```

Comment:
>unknown [](http://example.com/codeflow?start=25&length=7)

If the workspaceconfiguration isn't a map should we instead have a conversion utility than cast to unknown?

Reply:
This is test code and as such if we don't get a map returned, we have a bug and the test will fail. Leaving it as is will show the true nature of the bug. Furthermore, we are not changing the code or the logic here. The double cast was necessitated by stricter semantic rule enforcement in the new version.